js/src/components/google-mc-account-card/create-account-button.js

```diff
@@ -23,7 +23,9 @@ const CreateAccountButton = ( props ) => {
 	const { onCreateAccount = noop, onClick = noop, ...rest } = props;
 	const [ activeModal, setActiveModal ] = useState( MODALS.NONE );
 	const { data: existingAccounts } = useExistingGoogleMCAccounts();
-	const matchingDomainAccount = getMatchingDomainAccount( existingAccounts );
+	const matchingDomainAccount = getMatchingDomainAccount(
+		existingAccounts ?? []
+	);
```

Comment:
Wonder why this change is needed.

Reply:
Reverted the change, unless there was something specific that I might be missing @ankitrox ?